android/expoview/src/main/java/host/exp/exponent/utils/ExperienceActivityUtils.java

```diff
@@ -134,19 +136,43 @@ public static void configureStatusBar(@NonNull JSONObject manifest, final Activi
 
       setTranslucent(statusBarTranslucent, activity);
 
-      if (statusBarBackgroundColor == null || !ColorParser.isValid(statusBarBackgroundColor)){
-        // if backgroundColor is invalid or not set then it has to be transparent
-        setColor(Color.TRANSPARENT, activity);
-      } else {
-        setColor(Color.parseColor(statusBarBackgroundColor), activity);
+      String appliedStatusBarStyle = STATUS_BAR_STYLE_LIGHT_CONTENT;
+      if (statusBarStyle != null) {
+        appliedStatusBarStyle = setStyle(statusBarStyle, activity);
       }
 
-      if (statusBarStyle != null) {
-        setStyle(statusBarStyle, activity);
+      // Color passed from manifest is in format '#RRGGBB(AA)' and Android uses '#AARRGGBB'
+      String normalizedStatusBarBackgroundColor = RGBAtoARGB(statusBarBackgroundColor);
+
+      if (normalizedStatusBarBackgroundColor == null || !ColorParser.isValid(normalizedStatusBarBackgroundColor)) {
+        // backgroundColor is invalid or not set
```

Comment:
@bbarthec - can you ensure that expo-cli will warn people if the given background color is invalid?

Reply:
Opened new issue for this one https://github.com/expo/expo/issues/7322